internal/moreos/moreos_test.go

```diff
@@ -155,6 +157,65 @@ func Test_EnsureProcessDone(t *testing.T) {
 	require.NoError(t, EnsureProcessDone(cmd.Process))
 }
 
+func Test_EnsureChildProcessDone(t *testing.T) {
+	// There are 3 processes spawned by this test target, with the following relationship:
+	//
+	// runner (go run testdata/main/main.go)
+	// └── main (the executable, compiled from testdata/main/main.go. This is a fake func-e)
```

Comment:
ps I will make comments like this on any and all comments, so going to do this here. In all naming and organization things, simplest first. If there are no subdirs needed (ex main/) don't add one. If you have to say X is Y, call it Y. This helps make the cognitive associations fast, and ends up with less lines and words per line to convey the same thing

make sense?

```suggestion
	// └── fake_func-e (the executable, compiled from testdata/fake_func-e.go)
```

Reply:
Got it. Thanks!